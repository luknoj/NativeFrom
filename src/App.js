
import React, {Component} from 'react';
import {Platform, StyleSheet, Text, View} from 'react-native';
import { Header } from './components/common';
import Form from './components/Form';

export default class App extends Component {
  render() {
    return (
      <View>
        <Header title='Form' />
        <Form />
      </View>
    );
  }
}

