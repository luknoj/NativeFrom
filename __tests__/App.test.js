import React from 'react';
import App from '../src/App';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const app = renderer.create(<App />).toJSON();

  expect(app).toMatchSnapshot();
});