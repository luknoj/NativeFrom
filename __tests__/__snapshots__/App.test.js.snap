// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 60,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 16,
          "fontWeight": "600",
        }
      }
    >
      Form
    </Text>
  </View>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "margin": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000",
            "flex": 1,
            "fontSize": 16,
            "marginLeft": 10,
            "marginRight": 10,
          }
        }
      >
        Email
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        maxLength={100}
        onChangeText={[Function]}
        placeholder="jan.kowalski@wp.pl"
        style={
          Object {
            "flex": 2.5,
            "fontSize": 16,
            "marginRight": 10,
            "paddingBottom": 0,
            "paddingTop": 0,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "red",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "margin": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000",
            "flex": 1,
            "fontSize": 16,
            "marginLeft": 10,
            "marginRight": 10,
          }
        }
      >
        ImiÄ™
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        maxLength={20}
        onChangeText={[Function]}
        placeholder="Jan"
        style={
          Object {
            "flex": 2.5,
            "fontSize": 16,
            "marginRight": 10,
            "paddingBottom": 0,
            "paddingTop": 0,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "red",
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 40,
          "margin": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000",
            "flex": 1,
            "fontSize": 16,
            "marginLeft": 10,
            "marginRight": 10,
          }
        }
      >
        Kod pocztowy
      </Text>
      <TextInput
        allowFontScaling={true}
        autoCorrect={false}
        maxLength={6}
        onChangeText={[Function]}
        placeholder="12-345"
        style={
          Object {
            "flex": 2.5,
            "fontSize": 16,
            "marginRight": 10,
            "paddingBottom": 0,
            "paddingTop": 0,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "red",
        }
      }
    />
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "red",
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "#33b5e5",
          "borderRadius": 5,
          "marginTop": 10,
          "opacity": 1,
          "padding": 10,
          "width": 100,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "center",
            "color": "#fff",
            "fontSize": 16,
            "fontWeight": "600",
          }
        }
      >
        Submit
      </Text>
    </View>
  </View>
</View>
`;
